public/scripts/browse.js

```diff
@@ -0,0 +1,42 @@
+(function() {
+  var mapDiv = document.getElementById('myMap');
+
+  // set up map config - get mapbox token and location
+  var mapBoxToken = mapDiv.dataset.token;
+
+  if (!navigator.geolocation) {
```

Comment:
I know it might seem like overkill but it would really help me to have a comment illustrating that L7-L13 set the default location of the map based from browser location